en/99-apendice/1-upp.md

```diff
@@ -0,0 +1,110 @@
+## UPP
+UPP (Unified Principles and Practices) is an attempt to consolidate the good practices we have gathered for the development of projects throughout all these years. Like everything else in Manas, and this handbook in particular, it is far from being written in stone and is entirely dynamic; should you have a suggestion on how to improve the process, just send a pull request with the changes.
+
+UPP's format is a set of checklists that the project lead and the developers must verify at different stages throughout the project. To ensure it is brought into our daily working culture, we propose to remind each other these items as frequently as is needed.
+
+### Artifacts
+
+_Artifacts should be ready since the beginning of the project, and stay up-to-date whenever an issue is closed._
+
+#### Code repository
+* Follow **git-flow** as branching and tagging policy.
+* Follow the [repository rules](./2-repository-rules.md) guidelines.
+* Gather devtools and scripts useful for the project in a single `/tools` folder in the repository, with a brief explanation on the motivation and the usage of the tool.
+
+#### Initial repository files
+* **README.md**: include a brief explanation of the project (keeping in mind that it is the first thing anyone sees when opens the repository), how to set up a development environment, and a minimal description of its architecture.
+* **LICENSE**: choose an appropriate license for the project and ensure it is clear.
+* **CHANGELOG.MD**: keep a [changelog](http://keepachangelog.com/) up to date as each issue is closed, so that a _user_ of the system can understand the changes in each version.
+
+#### Issue tracker
+* **Milestones**: keep a set of future milestones with a description of their main goal and the epics it contains, noting that it might (actually should) change as the project moves on; this acts as a project's roadmap.
+* **States**: define the states that compose a ticket's lifecycle, and the action items that trigger a transition between states.
+* **Versioning**: keep a versioning policy following the repository rules.
+
+#### Design repository
+* **InVision** (or similar) to store the revisions of the project's design.
+* **Design per milestone**: after designing the design goal for a major release, work together with the design time to define the design for all intermediate versions that might reach production or be reviewed by an external audience; associate these intermediate UX versions to the roadmap's milestones, and always design them considering the current state of the application.
```

Comment:
I'd say: 'work together with the design team to define the screens for all intermediate versions that..'